lib/generators/exception_hunter/create_users/create_users_generator.rb

```diff
@@ -24,5 +24,19 @@ def install_devise
     def create_admin_user
       invoke 'devise', [name]
     end
+
+    def remove_registerable_from_model
+      return if options[:registerable]
+
+      model_file = File.join(destination_root, 'app', 'models', "#{file_path}.rb")
+      gsub_file model_file, /\:registerable([.]*,)?/, ''
+    end
```

Comment:
You can just invoke the devise installer without `registerable` I think

Reply:
is that so?